avm/src/vm/opcode.rs

```diff
@@ -1,79 +1,34 @@
 use futures::future::{join_all, FutureExt};
-use futures::task::{Context, Poll};
 use std::collections::HashMap;
 use std::convert::Infallible;
 use std::fmt::Debug;
 use std::future::Future;
 use std::hash::Hasher;
-use std::io::{self, Write, BufReader};
-use std::net::SocketAddr;
+use std::io::{self, Write};
 use std::pin::Pin;
 use std::str;
 use std::sync::Arc;
 use std::time::Duration;
 
-use async_stream::stream;
 use base64;
 use byteorder::{ByteOrder, LittleEndian};
 use dashmap::DashMap;
-use futures_util::stream::Stream;
 use hyper::header::{HeaderName, HeaderValue};
-use hyper::service::{make_service_fn, service_fn};
-use hyper::{client::{Client, HttpConnector, ResponseFuture}, server::Server, Body, Request, Response, StatusCode};
-use hyper_rustls::HttpsConnector;
+use hyper::{client::ResponseFuture, Body, Request, Response, StatusCode};
 use once_cell::sync::Lazy;
 use regex::Regex;
-use rustls::internal::pemfile;
-use tokio::net::{TcpListener, TcpStream};
 use tokio::process::Command;
 use tokio::sync::watch::{self, Sender, Receiver};
 use tokio::time::sleep;
-use tokio_rustls::TlsAcceptor;
-use tokio_rustls::server::TlsStream;
 use twox_hash::XxHash64;
 
+use crate::make_server;
```

Comment:
It's weird that the path to this macro is not `crate::vm::http::make_server` @cdmistman any ideas how to make it make sense, or is this the Rust Way?

Reply:
Thanks @cdmistman :)